pkg/labels/selector.go

```diff
@@ -229,21 +232,22 @@ func (r *Requirement) String() string {
 }
 
 // Add adds a requirement to the selector. It copies the current selector returning a new one
-func (lsel LabelSelector) Add(key string, operator Operator, values []string) Selector {
-	var reqs []Requirement
+func (lsel internalSelector) Add(key string, operator Operator, values []string) Selector {
```

Comment:
This should take a Requirement as an argument. I really don't like that it swallows errors silently.